runtime/queries/solidity/highlights.scm

```diff
@@ -244,10 +232,9 @@
   "new"
 ] @keyword.operator
 
-; TODO: move to top when order swapped
 ; identifiers
 ; -----------
-((identifier) @variable.builtin
- (#match? @variable.builtin "^(this|msg|block|tx)$"))
+((identifier) @variable.builtin (#match? @variable.builtin "^(this|msg|block|tx)$"))
```

Comment:
```suggestion
((identifier) @variable.builtin (#any-of? @variable.builtin "this" "msg" "block" "tx"))
```

when this was originally written I don't think we supported `#any-of?` but now we can replace the `#match?` with the equivalent `#any-of`